### PR TITLE
#1337: Tests Iterator Reverse

### PR DESCRIPTION
**Pull request by marceloamadeu**
This PR is for #1337

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1361](https://github.com/yegor256/cactoos/issues/1361) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/4d07939d-524a-47c4-aa70-519f282098ac); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/iterator/ReversedTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    public void reversesIterator() {
+        new Assertion<>(
+            "Must reverse the iterator",
+            new ListOf<>(
+                new Reversed<>(
+                    new IteratorOf<>("c", "a", "c", "t", "o", "o", "s")
+                )
+            ),
+            new IsEqual<>(
+                new ListOf<>(
```
Comment (victornoel) [inline src/test/java/org/cactoos/iterator/ReversedTest.java:51]: @marceloamadeu can't we directly compare iterators? or maybe use an existing matcher that allows to compare iterator?

Comment (marceloamadeu) [reply src/test/java/org/cactoos/iterator/ReversedTest.java:51]: @victornoel I couldn't find another way to resolve it. Do you have any suggestion?
Thank!!!

Comment (victornoel) [reply src/test/java/org/cactoos/iterator/ReversedTest.java:51]: @marceloamadeu let's forget about it, thank you for trying :)

**Comment by victornoel** [comment]
@paulodamaso good to merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 33 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@victornoel please make sure you will find at least 3 issues during next CR, thanks :)

**Comment by victornoel** [comment]
@sereshqua yes, thx

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**